.github/workflows/windows-build-xpu.yml

```diff
@@ -0,0 +1,128 @@
+name: windows-build-xpu
+
+on:
+  push:
+    tags:
+      - ciflow/xpu/*
```

Comment:
Please move the xpu windows build as a job in https://github.com/pytorch/pytorch/blob/main/.github/workflows/xpu.yml

Reply:
Let's check whether we can reuse existed callable workflow _win-build.yml and build scripts directly. If can't, we also need to modify the new added windows build workflow as a callable workflow